examples/cpp/infer_multiple_networks/inference_engine.cc

```diff
@@ -40,28 +40,29 @@ extern tf::Status LoadGraph(const string& graph_file_name,
                             std::unique_ptr<tf::Session>* session,
                             const tf::SessionOptions& options);
 
-extern tf::Status ReadTensorFromImageFile(const string& file_name,
+extern tf::Status ReadTensorFromImageFile(const std::vector<string>& file_names,
                                           const int input_height,
                                           const int input_width,
                                           const float input_mean,
                                           const float input_std, bool use_NCHW,
+                                          const int input_channels,
                                           std::vector<tf::Tensor>* out_tensors);
 
 namespace infer_multiple_networks {
 
 InferenceEngine::InferenceEngine(const string& name, const string& backend)
     : m_name(name) {}
-Status InferenceEngine::Load(const string& network, const string& image_file,
-                             int input_width, int input_height,
-                             float input_mean, float input_std,
-                             const string& input_layer,
+Status InferenceEngine::Load(const string& network,
+                             std::vector<string>& image_files, int input_width,
+                             int input_height, float input_mean,
+                             float input_std, const string& input_layer,
                              const string& output_layer, bool use_NCHW,
-                             bool preload_images) {
+                             bool preload_images, const int input_channels) {
```

Comment:
Minor
in this function signature some things are marked const others not. eg "input_channels" is marked const.
maybe make everything that can be: `const T&`